.github/workflows/test.yml

```
@@ -0,0 +1,37 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 1 * * 1" # At 01:00 (UTC) / 10:00 (JST) on Monday
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.7]
+        node: [16, 17, 18]
+    env:
+      BUNDLE_GEMFILE: test.gemfile
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - run: bundle install
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: "npm"
+      - run: npm install
+      - run: npm run build
+      - run: npm run start &
```

Maybe this is unnecessary
```suggestion
      - run: npm run start
```